crates/hulk/build.rs

```diff
@@ -12,7 +16,7 @@ fn main() -> Result<()> {
         cyclers: vec![
             CyclerManifest {
                 name: "Vision",
-                kind: CyclerKind::Perception,
+                kind: Perception,
```

Comment:
Do we really remove the enum prefix? For me it is more natural to have the enum type as prefix. Rare exceptions are some of Rust's prelude enum variants.

Reply:
Considering it says "kind" right next to it already, I think its more readable this way.

Reply:
But think about how others might use it if it is not used already in your scope. Maybe they rather think in my mentioned "more natural" way and will use `CyclerKind` which results in inconsistent style. Yes, you are right, that this is shorter, but I would weight the risk of future inconsistency higher.

Reply:
Not quite convinced but done.